local-ui-dev/README.md

```diff
@@ -0,0 +1,108 @@
+# Local UI Development
+
+This directory contains tools and instructions for using the RE api in a local development context which allows close operation with other kbase services in a custom docker network.
+
+This is enabled by running on the docker network `kbase-dev`.
+
+The `docker-compose.yaml` in this directory takes care of that.
+
+In addition, the configuration ensures that the entire schema set is loaded into Arango. This facilitates the usage of any collection during development.
+
+You'll need to take care of data loading.
+
+## Build and start the image
+
+From the repo root:
+
+```bash
+make start-ui-dev
+```
+
+## Install your data
+
+This is just an example of how one can load data using an importer, with the simple example of `data_sources`.
+
+Set up a Python environment:
+
+```bash
+virtualenv venv
+source venv/bin/activate
+```
+
+Install Python dependencies:
+
+```bash
+pip install -r dev-requirement.txt
+```
+
+Run your data importer. Each importer is an independent script, but they will probably follow a pattern similar to this example.
+
+This example uses the `data_sources.json` import data to populate the `data_sources_nodes` collection.
+
+Here we install from the default built-in data source, in dry-run mode, which prevents actual Arango changes:
+
+```bash
```

Comment:
It seems like there is a lot of redundant documentation between this file and the README found under data sources